pages/_error.js

```diff
@@ -1,17 +1,22 @@
 import { Button } from '../src/components/Button'
 
+import { SEO } from '../src/components/SEO'
+
 const reload = () => window.location.reload()
 const defaults = {
 	code: 'BEEP_BOOP_ERROR',
 	error: 'Something went wrong, sorry... 🥺'
 }
 
 //FIXME: always 200 ![FIXED?]
+//TODO: Seems to be giving right statusCode other than 200 yet to need verify for all possible scenarios
```

Comment:
Removing both comments would be better, less clutter in the dev environment when searching for TODOs or FIXME.